.ci/eslint-plugin-zotero-translator/lib/translators.js

```diff
@@ -143,15 +145,15 @@ function tryFormatJSON(raw) {
 		return raw;
 	}
 }
-function strip(source) {
+function strip(source, newline) {
```

Comment:
I'm getting some `undefined` inserted into the translators when fixing. I'm pretty sure that's because the `stip` function is also called from within `teslint`, but only with one argument and then `newline` is undefined in the replace below. Sorry for not trying to fix this, but I don't know this code at all so it'd take me a long time and I'm guessing it's a quick fix for you